docs/src/modules/operations/pages/observability-and-monitoring/view-logs.adoc

```diff
@@ -32,7 +32,7 @@ Logs can be exported for searching, reporting, alerting and long term storage by
 
 == Correlating logs
 
-You can correlate your log statements, those that you write in your application, by adding the MDC pattern `%mdc{trace_id}` to your log file when tracing is xref:operations:observability-and-monitoring/observability-exports.adoc#activating_tracing[enabled]. Like the following:
+You can correlate your log statements, those that you write in your application, by adding the MDC pattern `%mdc\{trace_id}` to your log file when tracing is xref:operations:observability-and-monitoring/observability-exports.adoc#activating_tracing[enabled]. Like the following:
```

Comment:
remove this warning: `WARN (asciidoctor): skipping reference to missing attribute: trace_id`